tests/osquery/test_api_views.py

```diff
@@ -44,6 +47,32 @@ def force_enrollment(self, tag_count=0):
             Enrollment.objects.create(configuration=configuration, secret=enrollment_secret),
             tags
         )
+    
+    def force_pack(self):
+        name = get_random_string(12)
+        return Pack.objects.create(name=name, slug=slugify(name))
+
+    def force_query(self, pack_query_mode=None, compliance_check=False):
+        if compliance_check:
+            sql = "select 'OK' as ztl_status;"
+        else:
+            sql = "SELECT * FROM osquery_schedule;"
+        query = Query.objects.create(name=get_random_string(12), sql=sql)
+        if pack_query_mode is not None:
+            pack = self.force_pack()
+            if pack_query_mode == "diff":
+                PackQuery.objects.create(
+                    pack=pack, query=query, interval=60, slug=slugify(query.name), log_removed_actions=False,
+                    snapshot_mode=False)
+            elif pack_query_mode == "snapshot":
+                PackQuery.objects.create(
+                    pack=pack, query=query, interval=60, slug=slugify(query.name), log_removed_actions=False,
+                    snapshot_mode=True)
+            else:
+                raise ValueError("Invalid pack_query value: {}".format(pack_query_mode))
```

Comment:
You can remove that because we control the test env, so a hard crash is OK. Less code.